modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -12771,6 +12771,7 @@ path-and-file-name=Path and File Name
 path-can-not-be-empty=Path cannot be empty.
 path-parameter-description-cannot-be-set-with-empty-path-parameter-property=Path parameter description cannot be set with empty path parameter property.
 path-parameter-must-be-an-external-reference-code,-id,-or-unique-field=Path parameter must be an external reference code, ID, or unique field.
+path-parameters-are-not-supported-by-api-endpoints-of-the-post-type=Path parameters are not supported by API endpoints of the POST type.
```

Comment:
I think that we could rename this like:
`Path parameters are not supported in POST API endpoints.`